jq_dict_parser.go

```diff
@@ -0,0 +1,45 @@
+package opslevel_jq_parser
+
+import (
+	"fmt"
+
+	libjq_go "github.com/flant/libjq-go"
+	"github.com/rs/zerolog/log"
+)
+
+type JQDictParser map[string]JQFieldParser
+
+func NewJQDictParser(dict map[string]string) map[string]JQFieldParser {
+	output := make(map[string]JQFieldParser)
+	if dict == nil {
```

Comment:
Looks like
```go
if dict == nil {
    return output
}
```
can be removed? 
Since dict is a `map[string]string` and its zero value would be any empty mapping, rather than `nil`

Reply:
@davidbloss AFAICT the zero value of a map would be nil - [source](https://go.dev/tour/moretypes/19).

Reply:
Good source, I copied it and added some prints in [this go playground](https://go.dev/play/p/v0P-c1HT17s) and this shows that we can remove this if statement

Reply:
https://go.dev/play/p/EF7c1sbVMzt You had a point here, since we are never assigning into the map a nil pointer exception would not have happened. NPE will only happen if assigning to map. Still, it does not hurt to keep the check.